compiler/luci/pass/src/MakeBatchNormGammaPositivePass.cpp

```diff
@@ -40,27 +40,39 @@ bool negative_gamma_to_positive(luci::CircleConst *gamma)
 }
 
 // Check if add is batchnorm add
-bool is_batchnorm_add(const luci::CircleAdd *add)
+bool is_batchnorm_add(const luci::CircleAdd *add, luci::CircleMul *&mul, luci::CircleConst *&beta)
```

Comment:
I guess that this will be replaced with the one in `FindPatternHelper.h` (https://github.com/Samsung/ONE/pull/6103). Do you want to merge both and update this file later?

Reply:
This is to minimize the change. I'll revise this again after #6103 lands.